.gitignore

```diff
@@ -13,7 +13,6 @@
 *.dylib
 
 # Generated by Cargo
-bin/
```

Comment:
I think it's out-dated. Cargo doesn't generate files in directories called `bin` any more, and this rule was leading to `test/src/bin/ws_server.rs` being excluded.